src/components/Link/Link.stories.tsx

```diff
@@ -41,14 +41,18 @@ export const backwardsLink = () => (
 );
 
 export const actionLinkWithDownloadIcon = () => (
-    <Link href="#passed-in-link" linkType={LinkTypes.Action}>
-        <Icon
-            name="download"
-            blockName="more-link"
-            modifiers={["left"]}
-            decorative={true}
-            iconRotation={IconRotationTypes.rotate0}
-        ></Icon>
-        Download
+    // To Pass in an icon and its link, make sure that the link tag wraps the icon.
+    // The passed-in content must be properly styled.
```

Comment:
Man, I super don't like this, this is the whole point of the DS—can we maybe apply flex to all subchildren instead of L45+L47?

Reply:
seems like there was an underlying issue here that I fixed.  This should not be necessary anymore.

Reply:
Let's make sure to remove the comment, then